codes/models/alt_growth_rates/IRN_adm1.do

```diff
@@ -172,27 +173,34 @@ graph drop hist_irn qn_irn
 
 // predicted "actual" outcomes with real policies
 *predict y_actual if e(sample)
-predictnl y_actual = p_1*_b[p_1] + p_2* _b[p_2] + p_1_x_Tehran*_b[p_1_x_Tehran] + p_2_x_Tehran*_b[p_2_x_Tehran] ///
-+ _b[_cons] + __hdfe1__ + __hdfe2__ if e(sample), ci(lb_y_actual ub_y_actual)
+predictnl y_actual = ///
+p_1 * _b[p_1] + ///
+p_2 * _b[p_2] + ///
+p_1_x_Tehran * _b[p_1_x_Tehran] + ///
+p_2_x_Tehran * _b[p_2_x_Tehran] + ///
+p_3 * _b[p_3] + ///
+_b[_cons] + __hdfe1__ + __hdfe2__ if e(sample), ci(lb_y_actual ub_y_actual)
 lab var y_actual "predicted growth with actual policy"
 
 // estimating magnitude of treatment effects for each obs
 gen treatment = ///
-p_1*_b[p_1] + ///
-p_2* _b[p_2] + ///
-p_1_x_Tehran*_b[p_1_x_Tehran] + ///
-p_2_x_Tehran* _b[p_2_x_Tehran] ///
+p_1 * _b[p_1] + ///
+p_2 * _b[p_2] + ///
+p_1_x_Tehran * _b[p_1_x_Tehran] + ///
+p_2_x_Tehran * _b[p_2_x_Tehran] + ///
+p_3 * _b[p_3] ///
 if e(sample)
 
 // predicting counterfactual growth for each obs
 *gen y_counter = y_actual - treatment if e(sample)
 predictnl y_counter =  _b[_cons] + __hdfe1__ + __hdfe2__ if e(sample), ci(lb_counter ub_counter)
```

Comment:
same thing here, should add testing regime

Reply:
fixed in all countries